wireless_modules/README.md

```diff
@@ -31,12 +31,23 @@ You will need to create a local version of `config.py` (`cp src/config.example.p
 The ESP32 needs to be flashed with the base micropython firmware. This only needs to be done once. See the [MicroPython ESP32 Getting Started guide](https://docs.micropython.org/en/latest/esp32/tutorial/intro.html) for detailed instructions.
 
 ## Uploading Code
-
+The `upload.sh` file in `middle_module` folder can be used to upload all relevant files on the ESP32.
 ```
 # Change this to the serial port of the ESP32
 ./upload.sh --port /dev/ttyUSB0
 ```
 
+## Running code on the ESP32
+If using `picocom`:
+1) Open terminal and connect the ESP32 to your computer
+2) Run `picocom <insert_device_port_number> -b 115200`. This would run `boot.py` and then `main.py`.
```

Comment:
So what you see by default is `>>>`? It works for me (the files run right after I use the `picocom` command), but from my experience I noted that this method only runs `boot.py` and `main.py` when you first use `picocom` after connecting the ESP32, after that you have to go into Python REPL and run `import main.py`. 
<img width="1141" alt="Screen Shot 2021-01-13 at 9 05 27 pm" src="https://user-images.githubusercontent.com/63642262/104438540-38df6100-55e4-11eb-84b9-292d5f80358d.png">

Reply:
Correction: Just tried it again today, and `picocom` command automatically ran the two files in multiple occasions, so it's not that it only works the first time you use it after connecting the ESP32 board. Maybe there's something else going on which I can't sniff out.

Reply:
That's weird. For me, `boot.py` and `main.py` will automatically run on power up i.e. when it's plugged in, but running `picocom` doesn't actually do anything.

So if you
- Plug in the ESP32
- Run `picocom`, and stop the program with control+c
- Exit `picocom`
- Run `picocom` again
The program runs automatically?

Reply:
Yep, but only in some circumstances which is weird. Like on one day it would run both files even the second time I run `picocom` and on other days it would only run the two files the first time I run `picocom`.

And the two files `boot.py` and `main.py` also run for me automatically when the ESp32 is powered up (when plugged in).

I might do some more testing with different combinations and see if I can spot a pattern.